samples/multiple-namespaces/README.md

```diff
@@ -0,0 +1,80 @@
+# RabbitMQ Knative Eventing Multi-Namespaces Cluster Example
+
+## Prerequisites
+
+- Same as listed [here](../../../docs/source.md#prerequisites)
+
+## Overview
+
+What this demo shows is how to manage multiple namespaces with one single RabbitMQ Cluster instance by creating 2 Broker-Trigger topologies in different namespaces, but connected to the same RabbitMQ Cluster and sending events through them.
```

Comment:
I think this is another contrived example that doesn't need a whole set of samples. Being able to create Brokers/Triggers in different namespaces isn't that complex to warrant an entire example imo. I know this section already existed, but it may make sense to remove it.

Reply:
In this regard, I'm the ones that like more examples for new users than less. But we could discuss this on the next weekly and see what other folks thinks about cc @salaboy @xtreme-sameer-vohra @ikvmw